src/cookbook/design/themes.md

```diff
@@ -53,34 +88,45 @@ MaterialApp(
 );
 ```
 
-See the [`ThemeData`][] documentation to see all of
-the colors and fonts you can define.
+Most `ThemeData` objects include two properties.
```

Comment:
Related to the comment above, technically speaking all ThemeData objects have those two properties -- they just aren't always specifically assigned by the app developer when calling the constructor.

If this sentence is intended to make the point that textTheme and colorScheme are two of the most important and commonly used properties of ThemeData, that's a great thing to include. Consider just calling it out directly that way.